### ISSUE TITLE
Describe the pipeline

### ISSUE THREAD

**Issue by homonecloco** [quirozcj/affinity_hap#1]
Hi, 
I found really hard to understand in which orden you run the scripts. I realised that you have two pairs of scripts. The .sh files don't match the names that you have with your python files. I would like to see the progress between version 1, 2, 3... etc, but it is too late to have it tracked here. 

I have a couple of questions. 

1. What is the point of combining all the files before? this is just duplicating the huge data tables. If you don't want to do it in memory, you should do it in a single job where you save the table in a temporary folder, ideally in the executing node. And then do the second part of the analysis. Otherwise you will end up with a huge number of files. 
2. What is the logic behind? I find difficult to understand what you are subsetting and how you are doing this. Mostly because you don't have an example that you can run with a toy dataset in here. So I can't see the input files that you are using. To make the code review I don't expect to go and dig in the cluster to find your files. 

I'm going to add comments on the commits themselves, so you can see which line I'm referring to, but it would had been more useful to have the history of the files, to understand the logic of your optimisation.

**Comment by quirozcj** [comment]
Hola Ricardo,


1 . I combined the files before because they are so many. After combining them, I can subsample to run AP with a subset of them if needed. I'm sure there are more efficient ways to do this, so there is plenty of room for improvement after testing that it works to call haplotypes. Sorry about the mess. It would be great to have a chat about this.

**Comment by quirozcj** [comment]
Fro the question 2, there are some files in the "test_data" folder but now updated the scripts, so I'll need to update those files because it need more files as a parameters, "chromosome lengths" etc...
I just updated the script that run the affinity. The line that run the clustering is the # 23, and that the bash script with more parameters as input. Still catching up, but I'll put those files ASAP tomorrow ...

**Comment by homonecloco** [comment]
So, for Question 1. Do you have the script that you used to merge the tables? that should also e part of the package. Are we going to integrate this with IBSpy or not? If so you really should be adding the code there. 
Is this ready for me to check the code?

**Comment by quirozcj** [comment]
Yes, the script that combine the tables is "combine_by_windows.py", I'll be updating everything tomorrow and during the weekend. This will be ready to check for Monday. Is this OK? Thanks!!
